src/ResourceManager/Network/Commands.Network/NetworkWatcher/TestAzureNetworkWatcherConnectivityCommand.cs

```diff
@@ -113,6 +113,16 @@ public override void Execute()
 
             MNM.ConnectivityParameters parameters = new MNM.ConnectivityParameters();
 
+            if (string.IsNullOrEmpty(this.DestinationId) && string.IsNullOrEmpty(this.DestinationAddress))
```

Comment:
Please add a test (can be a unit test) that exercises the new code paths

Reply:
Already tested the new code path in Powershell. I don't think tests are necessary for these new change, it only throw exception on invalid input.

Reply:
Then how do we prevent later changes from regressing this fix?

Reply:
we definetely need to cover all scenarios

Reply:
Note: it is OK to file an issue to add test automation next release, just link the issue here and we can merge for this release